CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 ## Unreleased
 
 - Migrated to github actions
+- Remodel cookbook as library
+  - Attributes removed
+  - Recipes removed
+- DHCPv6 support
```

Comment:
This doesn't follow the [Keep A Changelog spec](https://keepachangelog.com/en/1.0.0/) and probably should be updated to reflect that. In addition it should be mentioned here that all of the databag support has been removed.